mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -7499,6 +7499,101 @@
 				}
 			]
 		},
+		{
+			"name": " root-proxy",
+			"item": [
+				{
+					"name": "GET proxies",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "673f9fc6-1870-4840-9d2a-8effbce3917f",
+								"exec": [
+									"pm.test(\"success test\", function() {",
+									"    pm.response.to.be.json;",
+									"});",
+									"",
+									"//Check that status is 200",
+									"pm.test(\"Status is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"//Check that content-type is application/vnd.api+json",
+									"pm.test(\"'Content-Type' is application/vnd.api+json\", function() { ",
+									"      pm.response.to.have.header(\"Content-Type\");",
+									"      pm.expect(pm.response.headers.get(\"Content-Type\")).to.be.equal(\"application/vnd.api+json\");",
+									"});",
+									"",
+									"pm.test(\"Response must have a json body\", function () {",
+									"    pm.response.to.be.ok;",
+									"    pm.response.to.be.withBody;",
+									"    pm.response.to.be.json; ",
+									"});",
+									"",
+									"let response = pm.response.json();",
+									"",
+									"//Check data array is of type providers if not null",
+									"if(response.data) {",
+									"    let len = response.data.length;",
+									"    if(len > 0){",
+									"        //Get first record",
+									"        let firstRecord = response.data[0];",
```

Comment:
is it a common approach to validate only 1st record?

Reply:
as I can see - yes. The same approach is used for getting titles, packages, providers collections.
<img width="961" alt="screen shot 2018-12-07 at 12 04 33 pm" src="https://user-images.githubusercontent.com/37537790/49641268-4e9ef700-fa18-11e8-82ef-9003359a94e2.png">

Reply:
ok, thk
approved